org.eclipse.xtext.ui.tests/tests/org/eclipse/xtext/ui/tests/editor/model/edit/XtextDocumentModifyTest.java

```diff
@@ -64,14 +64,14 @@ public Object exec(XtextResource state) throws Exception {
 	}
 
 	@Test public void testCommentsNotDuplicated() throws Exception {
-		String grammar = "grammar foo.Foo\n" 
-			+ "generate foo \"http://foo.net/foo\"\n" 
-			+ "Foo: // comment in Foo \n"
-			+ "// comment before Assignment\n"
-			+ "  bars+=/* comment in assignment */Bar // comment after assignment\n"
-			+ "// comment before keywod\n"
-			+ "'foo';\n" 
-			+ "Bar: 'bar';";
+		String grammar = text("grammar foo.Foo", 
```

Comment:
maybe add some 

// @formatter:off
// @formatter:on

around this to prevent formatting

Reply:
done